src/std/fs.ab

```diff
@@ -76,3 +78,45 @@ pub fun change_owner(user: Text, path: Text): Bool {
 
     return false
 }
+
+/// Finds all files or directories matching a file glob.  Escapes all
+/// characters in the passed-in glob apart from "*", "?" and "/", to
+/// prevent injection attacks.
+pub fun glob(path: Text): [Text]? {
+    path = replace_regex(path, "\([^*?/]\)", "\\\\\1")
```

Comment:
Lets extract the escaping logic into a separate private function that gets called in both the regular and multiple versions of the function.

Reply:
Good call.